README.md

```diff
@@ -1,13 +1,14 @@
 # OpenTelemetry Semantic Conventions for Java
 [![Continuous Build][ci-image]][ci-url]
+[![Maven Central][maven-image]][maven-url]
 
 Java code-generation for the [OpenTelemetry Semantic Conventions](https://github.com/open-telemetry/semantic-conventions).
 
 ## Published released
 
-Published releases are available on maven central.
+Published releases are available on maven central. Replace `{{version}}` with the latest released version:
 
-TODO(jack-berg): insert maven badge indicating latest available release
+![GitHub release (with filter)](https://img.shields.io/github/v/release/open-telemetry/semantic-conventions-java)
```

Comment:
Check out how this renders here: https://github.com/jack-berg/semantic-conventions-java/tree/post-release-1.21.0#published-released